### ISSUE TITLE
FYI, hotfix

### ISSUE THREAD

**Issue by timwis** [open-austin/data-portal#35]
Hey guys just wanted to make you aware of a recent jekyll update at github that broke JKAN's ability to build, so editing probably doesn't work in your fork. 

Relevant issue: https://github.com/timwis/jkan/issues/127

Most JKAN forks can simply use the update button in the maintenance section of the admin portal, but yours may have merge conflicts since you've extended JKAN, so [here are the necessary changes](https://github.com/timwis/jkan/commit/f41e86e3d35513870896d54a0ed693f474df351a) for the hotfix.

**Comment by amaliebarras** [comment]
Hey Tim! I appreciate that!

PANDA's authentication broke at some point in the last month or so, but when we learned about [data.world](https://data.world/), we decided to create an org page there and use that as our Community Data Portal. Timing is a funny thing 😒

I did get the build failure yesterday too when I went to update our site with a blurb about that, but changing the theme to [Minima](https://pages.github.com/themes/) fixed it for my purposes
